src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.scss

```diff
@@ -16,6 +17,7 @@
     text-shadow: rgba(0,0,0, 0.8) 0 0 8px;
     text-overflow: ellipsis;
     white-space: nowrap;
+    z-index: 11;
```

Comment:
z-index is necessary in this case, i understand the concern, however. 
It had to be set for it to be visible, since we've used this value elsewhere. It also needs to be below the Shelf index.